src/App.scss

```
@@ -139,14 +139,8 @@ form.src-c-uuid {
 
   .disabled-tile-with-tooltip {
     display: inline-block;
-
-    .provider-icon {
-      opacity: 0.5;
-    }
-
-    .redhat-icon {
-      opacity: 0.5;
-    }
+    // PatternFly Tile component handles disabled state styling via isDisabled prop
```

Stylelint (17.11.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)